### PR TITLE
Introducing "Plaiceholder" 🐟

### PR DESCRIPTION
**Pull request by joe-bell**
I'm excited to introduce the final rebrand for `next-placeholder`, which now includes:
- Support for other Node.js-powered build tools/libraries (with examples for Next.js and 11ty) 🏗
- **Pure CSS** blurred image placeholders ✨
- ["Studio" GUI (and upcoming API)](https://plaiceholder.co) for project sponsors 🖼

This concludes a month-long experimentation phase, and I now feel comfortable to bump to version 1 with no renames/rebrands in the future.

Thank you to everyone for your support on this journey so far, I hope you enjoy the new features!

For more details, see the updated [README](https://github.com/joe-bell/plaiceholder).

## Migration

To allow for build tool flexibility, automatic sourcing for Next.js images from `public` has moved to the separate `@plaiceholder/next`package. `getImage(<path-to-public-file>)` will convert the file to an image buffer.

> 💡 Pro tip: before copying over exisiting implementations, consider using the new `@plaiceholder/css` strategy

### Base64 (`next-placeholder`)

```diff
  import * as React from "react";
- import { getBase64 } from "next-placeholder";
+ import { getImage } from "@plaiceholder/next"
+ import { getBase64 } from "@plaiceholder/base64"

export const getStaticProps: GetStaticProps = async () => {
- const imgSrc = "/keila-joa.jpg";
- const imgBase64 = await getBase64(imgSrc);
+ const imgPath = "/keila-joa.jpg";
+ const imgSrc = await getImage(imgPath);
+ const imgBase64 = await getBase64(imgSrc);

  return {
    props: {
      imgBase64,
      imgSrc,
    },
  };
};
```

### Blurhash (`next-blurhash`)

```diff
  import * as React from "react";
- import { getBlurhash } from "next-blurhash";
+ import { getImage } from "@plaiceholder/next"
+ import { getBlurhash } from "@plaiceholder/blurhash"

export const getStaticProps: GetStaticProps = async () => {
- const imgSrc = "/keila-joa.jpg";
- const imgHash = await getBlurhash(imgSrc);
+ const imgPath = "/keila-joa.jpg";
+ const imgSrc = await getImage(imgPath);
+ const imgHash = await getBlurhash(imgSrc);

  return {
    props: {
      imgHash,
      imgSrc,
    },
  };
};
```

---

**TL;DR**

Micahel Scott in the Office: 'Snip, snap! Snip, snap! Snip, snap! You have no idea the physical toll that three refactors have on a person'

### CODE REVIEW

**Comment by brandonpittman** [comment]
@joe-bell Awesome work!
